src/lib/stepper/stepper.spec.ts

```diff
@@ -364,30 +421,47 @@ function assertPreviousStepperButtonClick(stepperComponent: MdStepper,
 }
 
 /** Asserts that step position is correct for animation. */
-function assertCorrectStepPosition(stepperComponent: MdStepper, fixture: ComponentFixture<any>) {
+function assertCorrectStepPosition(stepperComponent: MdStepper,
```

Comment:
Can we rename this to `assertCorrectStepAnimationDirection`? I'm not really sure what "position" refers to. Also we can determine the stepper automatically given the fixture and take in a optional direction (`'ltr'` or `'rtl'`) rather than a boolean so that we aren't always passing `dir === 'rtl'`

Reply:
What about automatically getting the stepper from the fixture (in this method and others)?

Reply:
@mmalerba stepper is obtained from fixture by css query (e.g `fixture.debugElement.query(By.css('md-horizontal-stepper')).componentInstance`). Since these test functions are used commonly for both horizontal and vertical stepper, the stepper would need to be obtained first based on different test cases and then passed to the function for testing.

It could be possible to pass in a variable that checks to see if it's supposed to be horizontal or vertical stepper component then get the stepper from the fixture accordingly, but I'm not sure if that would be cleaner than how it's currently done.